starfish/pipeline/features/spots/detector/__init__.py

```diff
@@ -23,6 +24,13 @@ def add_to_parser(cls, subparsers):
         spot_finder_group = subparsers.add_parser("detect_spots")
         spot_finder_group.add_argument("-i", "--input", type=FsExistsType(), required=True)
         spot_finder_group.add_argument("-o", "--output", required=True)
+        spot_finder_group.add_argument(
```

Comment:
Acceptance of a dots image is required for all spot finders, so I moved it from the Gaussian spot detector to the parent class.